js/components/contactList/contactList_controller.js

```diff
@@ -235,4 +236,24 @@ angular.module('contactsApp')
 		return $routeParams.uid;
 	};
 
+	ctrl.getFirstContact = function(contactId) {
```

Comment:
the method's name is a bit misleading. From the name I would expect it to return a contact, but apparently it doesn't really return anything but change another state. For the sake of readability I'd suggest to rename this accordingly, maybe to something like `selectFirstContact`? :)